mysql-test/suite/rocksdb/t/issue7187.test

```diff
@@ -0,0 +1,389 @@
+INSTALL PLUGIN keyring_file SONAME 'keyring_file.so';
+
+call mtr.add_suppression("Plugin keyring_file reported: 'File .*keyring' not found .*");
+call mtr.add_suppression("keyring_file initialization failure. Please check if the keyring_file_data points to readable keyring file or keyring file can be created in the specified location. The keyring_file will stay unusable until correct path to the keyring file gets provided");
+
+CREATE TABLESPACE tab02k_e ADD DATAFILE 'tab02k_e.ibd'  FILE_BLOCK_SIZE 02k ENCRYPTION='Y' ;
+CREATE TABLESPACE tab04k ADD DATAFILE 'tab04k.ibd'  FILE_BLOCK_SIZE 04k ENCRYPTION='N' ;
+CREATE TABLESPACE tab08k ADD DATAFILE 'tab08k.ibd'  FILE_BLOCK_SIZE 08k ENCRYPTION='N' ;
+CREATE TABLESPACE tab16k_e ADD DATAFILE 'tab16k_e.ibd'  FILE_BLOCK_SIZE 16k ENCRYPTION='Y' ;
```

Comment:
how did this tablespace creation succeed? Keyring seems to be not initialized due to missing keyring_file_data.